Assets/Features/Autopilot/Provider Experimental/AutopilotDebug.cs

```diff
@@ -0,0 +1,97 @@
+﻿using EdyCommonTools;
+using Perrinn424.AutopilotSystem;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AutopilotDebug : MonoBehaviour
+{
+    public Rigidbody rb;
+    public RecordedLap lap;
+
+    private NearestSegmentSearcher segmentSearcher;
+    private Path path;
```

Comment:
Here why do we have both Path and path? is this typo, or do we have same variable defined in two ways?